integration-tests/error_functions.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net/http"
+)
+// Colocar as variáveis no código publish.go?
```

Comment:
I think they could be here but I'd rather rename this file to `errors.go`.

Reply:
Done.